doc/specs/stdlib_hash_procedures.md

```diff
@@ -940,30 +940,30 @@ Subroutine
 
 ##### Argument
 
-`harvest`: Shall be a scalar integer variable of kind `INT32`. It is
+`harvest`: Shall be a scalar integer variable of kind `int32`. It is
 an `intent(out)` argument.
 
 ##### Note
 
-`ODD_RANDOM_INTEGER` is intended to generate seeds for
- `UNIVERSAL_MULT_HASH`. `ODD_RANDOM_NUMBER` uses Fortran's intrinsic
- `RANDOM_NUMBER` and the values returned can be changed by calling the
- intrinsic `RANDOM_INIT`.
- 
+`odd_random_integer` is intended to generate seeds for
+ `universal_mult_hash`. `ODD_random_number` uses Fortran's intrinsic
```

Comment:
@wclodius2 I couldn't find `ODD_RANDOM_NUMBER` in the source code. Should it be:
```suggestion
 `universal_mult_hash`. `odd_random_integer` uses Fortran's intrinsic
```

Reply:
Its discussion is a leftover from earlier versions of the code. Some no longer included algorithms required an odd random seed. Either you or I can eliminate the discussion of that procedure.

Reply:
Should it be eliminate from the code too? (probably better I do it before merging, to avoid conflicts)

Reply:
As I remember, this is needed for `universal_mult_hash`, which is still provided. So, I think it could remain on the code, right?

Reply:
Sorry, I was wrong. I did a grep on `odd_random_number` for the source code, and didn't find anything, when, of course, I should have done a grep on `odd_random_integer`, which finds multiple references in the code. In the documentation the references to `odd_random_number` should have been changed to `odd_random_integer` and otherwise retained. I can add them back in if necessary after the merge.

Reply:
@wclodius2 I changed `odd_random_integer` to `odd_random_number` and kept the text there. After resolving a conflict, it should be fine now.

Reply:
Odd, I had suggested changing `odd_random_nuber` to `odd_random_integer`, while you claim to have done the reverse. FWIW I have no problems with that if you also changed it in the source code.

Reply:
I am sorry. I meant "I changed `odd_random_number` to `odd_random_integer", as you proposed.
Again sorry for the confusion.

Reply:
Great!